### PR TITLE
Split receipt page into separate component

### PR DESCRIPTION
**Pull request by norbye**
With the old structure and sessionStorage handling you could start writing an application, delete it, start writing a new one - press cancel, and because of the sessionStorage it would display as if you had submitted your new application although you hadn't.

With this change, the receipt page is split into a separate component that should only display data fetched directly from the backend, and the sessionStorage is only used in the editing components.

Previously there was a field specifying whether the user is currently editing their application which enabled for the possibility of the final application not being displayed by default if the sessionStorage got desynchronized. This is changed to a timestamp for every sessionStorage update, which is compared to the timestamp of the submitted application to decide which view should be shown to the user.

sessionStorage is separated to utils to handle the usage better and enforce the timestamp.

### CODE REVIEW

File: frontend/src/routes/ReceiptForm/FormStructure.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+        <Title>Innsendt data</Title>
+      </FormHeader>
+      <Form>
+        <SeparatorLine />
+        <GeneralInfoSection>
+          <SectionHeader>Generelt</SectionHeader>
+          <HelpText>
+            <Icon name="information-circle-outline" />
+            Mobilnummeret vil bli brukt til å kalle deg inn på intervju av
+            komitéledere.
```
Comment (ivarnakken) [inline frontend/src/routes/ReceiptForm/FormStructure.js:138]: Should we add some info of how the phone number (and the "Innsendt data" in general) is treated during and after the admissions are over? Nevertheless out of scope of what you're trying to achieve with this PR.

File: frontend/src/routes/ReceiptForm/FormStructure.js [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+            disabled={true}
+          />
+
+          <HelpText>
+            <Icon name="information-circle-outline" />
+            Kun leder av Abakus kan se det du skriver inn i prioriterings- og
+            kommentarfeltet.
+            <Icon name="information-circle-outline" />
+            Det er ikke sikkert prioriteringslisten vil bli tatt hensyn til.
+            Ikke søk på en komite du ikke ønsker å bli med i.
```
Comment (ivarnakken) [inline frontend/src/routes/ReceiptForm/FormStructure.js:152]: `komite` -> `komité`

**Review by ivarnakken — APPROVED** [review]
This is much cleaner - good job! 💯 Just some minor preference comments

File: frontend/src/routes/ApplicationForm/index.js [hunk 3]
```diff
@@ -6,6 +6,10 @@ import { useCreateApplicationMutation } from "src/query/mutations";
 import GroupApplication from "src/containers/GroupApplication";
 
 import FormStructure from "./FormStructure";
+import {
+  getPhoneNumberDraft,
+  getPriorityTextDraft,
+} from "../../utils/draftHelper";
```
Comment (LudvigHz) [inline frontend/src/routes/ApplicationForm/index.js:12]: Absolute paths pls

**Review by LudvigHz — APPROVED** [review]

**Comment by norbye** [comment]
Rebasing with the changes from master had quite some conflicts, so if you guys have anything to note with the rebase let me know

**Review by eikhr — APPROVED** [review]

**Pull request merged**